cron/alerts/main.go

```diff
@@ -329,22 +230,34 @@ func logLastError(config AlertsConfig, lastError string, badRecipients []string)
 	log.Print(msg)
 }
 
-func handler(config AlertsConfig) error {
-	if err := config.init(); err != nil {
+func handler() error {
+	config, err := getConfigFromAWS()
+	if err != nil {
 		return err
 	}
 
-	cfRecords := getCFRecords(config)
+	for _, alertConfig := range config.Alerts {
+		log.Printf("Starting scan for alert config %q", alertConfig.Title)
+		cfRecords := getCFRecords(*config, alertConfig)
 
-	if len(cfRecords) < 1 {
-		log.Printf("\n No records found in Cloudflare containing any of these: %v", config.CFContainsStrings)
-		return nil
-	}
+		if len(cfRecords) < 1 {
+			log.Printf("\n No records found in Cloudflare containing any of these: %v", alertConfig.CFContainsStrings)
+			return nil
+		}
 
-	sendEmails(config, cfRecords)
+		sendEmails(*config, alertConfig, cfRecords)
+	}
 	return nil
 }
 
 func main() {
 	lambda.Start(handler)
 }
+
+func getEnv(key, defaultValue string) string {
```

Comment:
Partially wondering if it would be cleaner to change return value to either aws.string or *string, as that is the end result after calling this. It's fine as is too.

Reply:
Thanks, I prefer this way. I have a bias against using pointers unless necessary.